packages/cosmwasm-stargate/src/cosmwasmclient.spec.ts

```diff
@@ -275,9 +275,9 @@ describe("CosmWasmClient", () => {
       const expectedAddresses = deployedHackatom.instances.map((info) => info.address);
 
       // Test first 3 instances we get from scripts/wasmd/init.sh. There may me more than that in the result.
-      expect(result[0]).toEqual(expectedAddresses[0]);
+      expect(result[0]).toEqual(expectedAddresses[2]);
       expect(result[1]).toEqual(expectedAddresses[1]);
-      expect(result[2]).toEqual(expectedAddresses[2]);
+      expect(result[2]).toEqual(expectedAddresses[0]);
```

Comment:
I wonder if we really want to change the order here. It was a UI decision to sort from new to old in the code explorer. But here? Let's use the ordering we had before (ascending by creation time, the same way we get it from the node). The app can change the order if it wants to.